vite.config.ts

```diff
@@ -0,0 +1,61 @@
+import {readFileSync} from "node:fs";
+import {dirname, join, resolve} from "node:path";
+
+import {glob} from "fast-glob";
+import {defineConfig} from "vite";
+
+// Create aliases for each package in the Perseus monorepo, so Vite knows
+// where to look when a file imports e.g. @khanacademy/perseus.
+const packageAliases = {};
+glob.sync(join(__dirname, "/packages/*/package.json")).forEach(
+    (packageJsonPath) => {
+        const pkg = JSON.parse(readFileSync(packageJsonPath, "utf-8"));
+        packageAliases[pkg.name] = join(dirname(packageJsonPath), pkg.source);
+    },
+);
+
+export default defineConfig({
+    define: {
+        // This is used to determine if we are running in a
+        // Dev/Storybook environment.
+        "process.env.STORYBOOK": JSON.stringify(true),
```

Comment:
This is a funny way of spelling `"true"` 😂 

No change needed.

Reply:
Haha! True. It does quote it though... so there's that.

<img width="174" alt="image" src="https://github.com/user-attachments/assets/586f07f6-e372-46c0-9124-3bb2874f0180" />

Reply:
All it does it convert it to a string. The following two expressions are equivalent:

```ts
JSON.stringify(true);
"true";
```